packages/saved-views-client/src/client/ITwinSavedViewsClient.ts

```diff
@@ -33,40 +37,46 @@ export class ITwinSavedViewsClient implements SavedViewsClient {
     this.getAccessToken = args.getAccessToken;
   }
 
-  private async queryITwinApi<ReturnType>(queyParams: QueryParams): Promise<ReturnType> {
+  private async queryITwinApi<ReturnType>(args: QueryParams): Promise<ReturnType> {
```

Comment:
I'm just curious: Why is the renaming from query params to args? Isn't query params more descriptive?

Reply:
It looks like it may be for code consistency. If so feel free to resolve comment.

Reply:
`args` is shorter and there is no way to misinterpret the variable's purpose.